From e2d3677ec5fefc84792a330f87de769e8b3e9d9b Mon Sep 17 00:00:00 2001
From: Tristan Seligmann <mithrandi@mithrandi.net>
Date: Mon, 10 Oct 2016 04:40:30 +0200
Last-Update: Wed, 10 Oct 2018 09:57:30 +0200
Reviewed-by: Salvo 'LtWorf' Tomaselli <tiposchi@tiscali.it>
Subject: Point intersphinx_mapping at local Python docs.


Index: python-attrs-19.3.0/docs/conf.py
===================================================================
--- python-attrs-19.3.0.orig/docs/conf.py
+++ python-attrs-19.3.0/docs/conf.py
@@ -168,8 +168,9 @@ texinfo_documents = [
 ]
 
 intersphinx_mapping = {
-    "https://docs.python.org/3": None,
-    "https://zopeinterface.readthedocs.io/en/latest/": None,
+    'python': ('/usr/share/doc/python3-doc/html', '/usr/share/doc/python3-doc/html/objects.inv'),
+    # Zope Interface package does not build sphinxdoc at the moment
+    'zope.interface': ('/usr/share/doc/python-zope.interface-doc/html', '/usr/share/doc/python-zope.interface-doc/html/objects.inv'),
 }
 
 # Allow non-local URIs so we can have images in CHANGELOG etc.
