From: Tristan Seligmann <mithrandi@mithrandi.net>
Date: Mon, 10 Oct 2016 04:40:30 +0200
Subject: Point intersphinx_mapping at local Python docs.

---
 docs/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index ef5c419..83a5993 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -163,7 +163,7 @@ texinfo_documents = [
 epub_description = "Python Clases Without Boilerplate"
 
 intersphinx_mapping = {
-    "https://docs.python.org/3": None,
+    'python': ('/usr/share/doc/python3-doc/html', '/usr/share/doc/python3-doc/html/objects.inv'),
 }
 
 # Allow non-local URIs so we can have images in CHANGELOG etc.
